frontend/src/app/components/modals/export-modal/wp-table-export.modal.ts

```diff
@@ -63,6 +102,47 @@ export class WpTableExportModal extends OpModalComponent implements OnInit {
     });
   }
 
+  private triggerByLink(url:string, event:MouseEvent) {
+    event.preventDefault();
+
+    this.downloadSwitched(url);
+  }
+
+  private downloadSwitched(url:string) {
+    this
+      .triggerCall(url)
+      .then((data) => {
+        if (data.status === 202) {
+          this.pollUntilDownload(data.url!);
+        }
+      })
+      .catch((data) => {
+        this.errorOrDownload(data);
+      });
+
+    this.startLoadingIndicator();
+  }
+
+  private triggerCall(url:string) {
+    return this
+      .httpClient
+      .get(url, { observe: 'response' })
+      .toPromise();
+  }
+
+  private pollUntilDownload(url:string) {
+    let poller = this
+      .httpClient
+      .get(url, { observe: 'response'})
+      .pipe(
+        catchError(this.errorOrDownload.bind(this))
+      );
+
+    this.subscription = poller.pipe(
+      expand(_ => timer(1000).pipe(concatMap(_ => poller)))
+    ).subscribe();
```

Comment:
@oliverguenther there is still the console error at the end of a download:

![image](https://user-images.githubusercontent.com/617519/79584201-5ec7eb80-80ce-11ea-8a15-39edc73bb8c8.png)

As your Angular foo is stronger, maybe you can figure out the problem.